### ISSUE TITLE
Add support for Unifi PDU?

### ISSUE THREAD

**Issue by kashalls** [unpoller/dashboards#26]
There is already support for unpoller to fetch the metrics for the Unifi PDU Pro.
For prometheus, they are made available at `unpoller_device_outlet_`.

Is it possible to have a dashboard made for the pdu?

**Comment by platinummonkey** [comment]
Certainly. I don’t own one myself to do anything meaningful, but I am happy to review PRs - attach a screenshot please!

**Comment by kashalls** [comment]
> Certainly. I don’t own one myself to do anything meaningful, but I am happy to review PRs - attach a screenshot please!

I haven't designed anything quite yet, I'm struggling to figure out a way that this would turn out nice and usable at this moment.

**Comment by davidnewhall** [comment]
We need someone with one of these devices to make us a nice panel for it.

**Comment by davidnewhall** [comment]
The USW dashboard has panels for watts, amps, etc on PoE ports. You can use those as an example.

**Comment by kashalls** [comment]
Well, it took me almost a whole year to get it done, but I finally got something I am happy with.

![3JPMfqA](https://github.com/user-attachments/assets/6e3002dd-ac85-4d85-9587-abf122f6b8f9)
![7K1doew](https://github.com/user-attachments/assets/5ce2ad6c-495e-4180-813d-aaa45ff151c9)

I am not sure how to set up the dynamic datasource, and this is only in Prometheus. I would need someone else to convert it to influxdb but I do not know how. Let me know if you have any questions, and I will make a PR later today.
